src/main/java/seedu/duke/ui/AppUi.java

```diff
@@ -157,8 +166,9 @@ public boolean matchCommand(String command, HashMap<String, String> commandArgum
             break;
 
         case "exit":
-            System.out.println(goodByeMessage);
-            return false;
+            // System.out.println(goodByeMessage);
+            // return false;
```

Comment:
Generally okay, maybe can remove commented out code

Reply:
oh sorry i missed that